src/work-allocation-2/guards/work-allocation-feature-toggle.guard.ts

```diff
@@ -0,0 +1,26 @@
+import { Injectable } from '@angular/core';
+import { CanActivate, Router } from '@angular/router';
+import { FeatureToggleService } from '@hmcts/rpx-xui-common-lib';
+import { tap } from 'rxjs/operators';
+
+import { AppConstants } from '../../app/app.constants';
+
+@Injectable()
+export class WorkAllocationFeatureToggleGuard implements CanActivate {
+  public static defaultUrl: string = '/cases';
+  constructor(private readonly featureToggleService: FeatureToggleService,
+              private readonly router: Router) {
+  }
+
+  public static navigateUrl(isfeatureEnabled: boolean, router: Router, url: string) {
+    if (!isfeatureEnabled) {
+      router.navigate([url]);
+    }
+  }
+
+  public canActivate() {
```

Comment:
_Nitpick:_
Should have a return type...

Reply:
Done